### ISSUE TITLE
persistent cache for result of src_organism()?

### ISSUE THREAD

**Issue by vjcitn** [Bioconductor/Organism.dplyr#4]
From the current vignette:

Running src_organism() without a given path will save the sqlite file to a tempdir():

...

It might be more convenient if the default behavior was to write the sqlite into a folder
like the one used for AnnotationHub, with src_organism checking for a relevant database
when invoked.  It takes a bit of time to build the database, and if I understand the default behavior correctly, it will be lost when the session ends.

**Comment by mtmorgan** [comment]
Yes, that's the intention when [BiocFileCache](https://hedgehog.fhcrc.org/bioconductor/trunk/madman/Rpacks/BiocFileCache) (@lshep) becomes available.
